config/webpack/environment.js

```diff
@@ -1,12 +1,16 @@
 const { environment } = require('@rails/webpacker')
+const webpack = require('webpack')
 
-let ASSET_HOST
-
-if (process.env.NODE_ENV === 'production') {
-  ASSET_HOST =
-    'https://exercism-assets-staging.s3.eu-west-2.amazonaws.com/packs/'
```

Comment:
@kntsoriano I feel like this shouldn't have been removed?

Reply:
Yes. We don't need this anymore.